contents/handbook/engineering/support-hero.md

```diff
@@ -64,9 +64,9 @@ As the support hero you should triage any bugs/feature requests that you don't d
 
 Supporting self-hosted users can be particularly tricky/time-consuming!
 
-If the messages are from a dedicated slack, a tag from the CS team or 'high' priority zendesk then **do prioritize** them accordingly. You'll likely find [these docs useful](https://posthog.com/docs/self-host/deploy/troubleshooting).
+If the messages are from a dedicated slack, a tag from the CS team or 'high' priority zendesk then these are high-priority and **do prioritize** them accordingly. You'll likely find [these docs useful](https://posthog.com/docs/self-host/deploy/troubleshooting).
 
-However, if it's in the community slack and you don't have the time to solve it then it's fine to politely point them to the docs for [self-serve open-source support](/docs/self-host/open-source/support#support-for-open-source-deployments-of-posthog) and ask them to file a github issue if they believe something is broken in the docs or deployment setup.
+However, if it's in the community slack then these are low priority. If you don't have the time to solve it then it's fine to politely point them to the docs for [self-serve open-source support](/docs/self-host/open-source/support#support-for-open-source-deployments-of-posthog) and ask them to file a github issue if they believe something is broken in the docs or deployment setup. *Note:* If it's legacy (signed up before December 2022) PostHog Scale user that's self-hosting and paying a large enough amount it's worth asking Simon/Cameron if we should setup a dedicated slack with them and/or move them to an enterprise plan.
```

Comment:
how would I know how much they are paying?
Can we do this the other way around - anyone who's currently paying we'll talk to and move to dedicated slack channel or not, everyone else we ignore and send a <scripted message>, e.g.

Looks like you're using our chart, unfortunately the high support load for our team was too high to continue supporting this option outside of our Enterprise offering <link>, alternatively you can switch to our cloud product <link> or <hobby deploy>

Reply:
This sounds right to me. We should assume everyone in users slack is not paying and explicitly prioritize support for slack connect channels. Then if we are missing people through the cracks (paying customers in users slack), we can solve that problem.

Reply:
There's a lot of current paying customers for Scale which we likely don't want to have dedicated Slack channels for - I'd not want to create a channel for all of them in favour of keeping the dedicated Slack connect channels for our priority customers so that support heroes know that if it's a dedicated Slack channel it's _important_
 
My preference would be to message in User Slack to say community support only as well as encourage people to move to Cloud (esp. now we have EU)

Reply:
Makes sense to me!